packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -1010,6 +1050,52 @@ export class BasePicker<T extends {}, P extends IBasePickerProps<T>>
     }
   }
 
+  private _getRequiredFieldErrorMessage(items: T[]): string | JSX.Element | undefined {
+    if (!this.props.required || (items && items.length > 0)) {
+      return;
+    }
+    return 'Please fill out this field.';
```

Comment:
We generally do not include strings in our components, since we don't handle localization. I think the default here would be to rely on `errorMessage` and `onGetErrorMessage` and not perform validation or render a default message.